usaspending_api/spending_explorer/v2/filters/type_filter.py

```diff
@@ -45,7 +47,7 @@ def get_unreported_data_obj(queryset, limit, spending_type, actual_total, fiscal
         values_list('total_obligation', flat=True). \
         first()
 
-    if spending_type in VALID_UNREPORTED_DATA_TYPES:
+    if spending_type in VALID_UNREPORTED_DATA_TYPES and list(filters.keys()).sort() == VALID_UNREPORTED_FILTERS.sort():
```

Comment:
Comparing equality between two `set`s might be better. A quick check in the REPL showed that sets do not need to be ordered for equality

`... and set(filters.keys()) == set(VALID_UNREPORTED_FILTERS):`

Reply:
Updated to use set comparison for completeness